### PR TITLE
Support all AKN elements in HTML: update XSL

### PR DESCRIPTION
**Pull request by goose-life**
See https://github.com/laws-africa/indigo-lawsafrica/issues/835

### CODE REVIEW

File: indigo_api/static/xsl/html_akn.xsl [hunk 1]
```diff
@@ -55,17 +55,41 @@
     </xsl:attribute>
   </xsl:template>
 
-  <xsl:template match="a:article">
+  <!-- containers with headings on the next line -->
+  <xsl:template name="container-1">
```
Comment (longhotsummer) [inline indigo_api/static/xsl/html_akn.xsl:59]: Minor: `container` has a specific meaning in AKN. Perhaps `hier-heading-next-line` or similar is better?

**Review by longhotsummer — DISMISSED** [review]
Approved, but I think the `container` template name could be less confusing.

**Comment by goose-life** [comment]
@longhotsummer I made a further change to the template to cut down in repetition hence the re-request

**Review by longhotsummer — APPROVED** [review]
great

**Pull request merged**